src/scp/SCP.cpp

```diff
@@ -120,28 +120,93 @@ SCP::getJsonInfo(size_t limit, bool fullKeys)
     return ret;
 }
 
+SCP::QuorumInfoNodeState
+SCP::getState(NodeID const& node, uint64 slotIndex, int N = 2)
+{
+    for (int k = 0; k < N; k++)
+    {
+        if (slotIndex - k <= 0)
```

Comment:
check should be written as `slotIndex <= k` (result is unsigned so what you did is invalid)

Reply:
Fixed.